src/components/Modals/DownloadDataModal/DownloadDataModalContainer.spec.js

```diff
@@ -30,4 +36,10 @@ describe('Component > DownloadDataModalContainer', function () {
     wrapper.props().onClose()
     expect(toggleModalSpy).toHaveBeenCalledWith('')
   })
+
+  it('should call toggleModal on child onClose', function () {
```

Comment:
It looks like this test might have been duped from the one above it. The test names are identical and there's a duped assertion (line 37 and line 43 are identical).

Can we change the name of this test to 'should call dataDownload when download clicked'?

Might also not need the assertion on line 43 (unless you also want to verify that the modal closes when you click download button).

Reply:
Good catch! Yes, I'd also like to assert the modal closes when the download button is clicked.

Reply:
The duped assertion is on purpose, as that is to check that the modal closes.